docs/README.md

```diff
@@ -109,6 +109,30 @@ After starting TWCManager, the script will run in the foreground and will regula
    * The line below this reports the same values but in amps instead of watts.
    * The final line shows the minAmpsPerTWC value, which is the minimum number of amps that the master must offer each slave before we tell the attached vehicle to charge (via the Tesla API).
 
+## Running TWCManager as a Service
+
+The following commands make TWCManager run automatically (as a service) when the Raspberry Pi boots up:
+
+Install the service
+```
+sudo systemctl enable /home/pi/TWCManager/twcmanager.service
```

Comment:
While this works, and usually is okay, this fails if /home is a separate partition. In that case you should copy the file, say:
`sudo cp /home/pi/TWCManager/twcmanager.service /etc/systemd/system/twcmanager.service`
After that you can run:
`sudo systemctl enable twcmanager.service --now`
That will enable the service to run on boot and (because of the --now flag) starts it immediately.